### PR TITLE
[RD] Add ACH Payment Method

### PR DESCRIPTION
**Pull request by npsp-reedestockton**
# Critical Changes

# Changes
 - We added `ACH` to the Payment Method field on Recurring Donations to support Elevate customers. This new value is meant to replace the existing `ACH/EFT` value. If you already upgraded to Enhanced Recurring Donations, you should take the following action:
   - Go to **Setup** and click **Object Manager**.
   - Find and click **Recurring Donation**.
   - Click **Fields & Relationships**.
   - Click **Payment Method**.
   - Click **Del** next to ACH/EFT.
   - Click **OK**.
   - For Replace value on records with, select `ACH`.
   - Click **Save**.

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by npsp-reedestockton** [comment]
Tracking [W-8206753](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008gsgpIAA/view)

**Comment by npsp-reedestockton** [comment]
@RadSFDeveloper @andrewyu-salesforce Ready for review.
@sam-knox Would you please review description?

**Review by sam-knox — APPROVED** [review]

**Review by RadSFDeveloper — COMMENTED** [review]
@npsp-reedestockton , new orgs will have both "ACH/EFT" and "ACH" values. We can either replace "ACH/EFT" with "ACH" or remove "ACH/EFT"  anywhere in the codebase and add "ACH". These are options 1 and 2 respectively that @force2b  mentioned.

**Comment by npsp-reedestockton** [comment]
@RadSFDeveloper original push was prior to your response that replacement should occur.  Changed code accordingly.

**Review by RadSFDeveloper — COMMENTED** [review]
There is a mentioned of ACH/EFT in `RD2_ScheduleService_TEST`. Also, should we replace "ACH/EFT" in translation files?

**Comment by npsp-reedestockton** [comment]
@RadSFDeveloper replaced all refs to ACH/EFT _except_ French translation, which is not literal.

File: src/objectTranslations/npe03__Recurring_Donation__c-fr.objectTranslation [hunk 1]
```diff
@@ -202,7 +202,7 @@
         <label>Mode de paiement</label>
         <name>PaymentMethod__c</name>
         <picklistValues>
-            <masterLabel>ACH/EFT</masterLabel>
+            <masterLabel>ACH</masterLabel>
             <translation>CCA/TEF</translation>
```
Comment (RadSFDeveloper) [inline src/objectTranslations/npe03__Recurring_Donation__c-fr.objectTranslation:206]: I am assuming that "CCA/TEF" can be replaced with "CCA" similarly to the Spanish translation removal.

Comment (npsp-reedestockton) [reply src/objectTranslations/npe03__Recurring_Donation__c-fr.objectTranslation]: Okay.  Pushed that change as well.

**Review by RadSFDeveloper — APPROVED** [review]
Looks good, thanks @npsp-reedestockton !

**Comment by RadSFDeveloper** [comment]
@sam-knox , do release notes need to indicate that "ACH/EFT" is replaced with "ACH" and that certain manual changes need to be done?

**Comment by sam-knox** [comment]
> @sam-knox , do release notes need to indicate that "ACH/EFT" is replaced with "ACH" and that certain manual changes need to be done?

@RadSFDeveloper - Yes, now that I know the plan I will include instructions. My thought was to encourage people to use the replace picklist value functionality when they deactivate "ACH/EFT".

**Pull request merged**